lib/routes/routes_page.dart

```diff
@@ -356,14 +58,32 @@ class _RoutesPageBodyState extends State<_RoutesPageBody> {
             padding: const EdgeInsets.only(bottom: 140),
             child: Consumer<FilterChangeNotifier>(
                 builder: (context, filterChangeNotifier, child) {
-              return Column(
-                  children: routes
-                      .where((route) => filterChangeNotifier.filters
-                          .every((filter) => filter.isValid(route)))
-                      .map((ClimbingRoute route) =>
-                          RouteListElement(route: route))
-                      .toList());
-            }))
+              return StreamBuilder<List<RouteData>>(
```

Comment:
Nitpick: Use shorthand return instead of {return ...}

Reply:
I think this way is more concise and recommended by dart linter [here](https://dart-lang.github.io/linter/lints/use_build_context_synchronously.html)

Reply:
I am referring to the return statement, not the if statement